api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package org.pmiops.workbench.firecloud;
+
+import javax.servlet.http.HttpServletResponse;
+import org.pmiops.workbench.firecloud.api.ProfileApi;
+import org.pmiops.workbench.firecloud.model.Me;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class FireCloudServiceImpl implements FireCloudService {
+
+  @Override
+  public boolean isRequesterEnabledInFirecloud() throws ApiException {
+    Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+    ProfileApi profileApi = new ProfileApi();
+    ApiClient apiClient = new ApiClient();
```

Comment:
Added FireCloudConfig which sets up an ApiClient binding that can be injected with a Provider here.